src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/group_convolution.cpp

```diff
@@ -289,6 +291,34 @@ std::vector<CPUSpecificParams> filterCPUInfoForDeviceSupportBF16(std::vector<CPU
     }
     return resParamsSet;
 }
+
+std::vector<CPUSpecificParams> filterCPUInfoForDeviceSupportBrdgmm(std::vector<CPUSpecificParams> CPUParams) {
```

Comment:
Why we cannot use already existing filterCPUInfoForDevice* functions?

Reply:
currently I meet an issue in test case, I also met this issue in https://github.com/openvinotoolkit/openvino/pull/19721/files#r1582591079 , the issue is that only **the max ISA configure**  can be tested on the test machine, for example I created configure brgconv_avx512_dw and brgconv_avx2_dw, I expected brgconv_avx2_dw  also can be tested on SPR machine, but this test case can't pass, because onednn always select brgconv_avx512_dw, the custom priority is not work in this situation. I think this is because that there is no ISA value on  **CPU_INSTANCE_AVX512(brdgmm_dw_convolution_fwd_t)**, it don't like **CPU_INSTANCE_AMX(brgemm_1x1_convolution_fwd_t<avx512_core_amx>** which has isa **avx512_core_amx**
```
        {{forward, f32, f32, f32}, {
            CPU_INSTANCE_AVX512(brdgmm_dw_convolution_fwd_t)
            CPU_INSTANCE_X64(ip_convolution_fwd_t)
            CPU_INSTANCE_AMX(brgemm_1x1_convolution_fwd_t<avx512_core_amx>)
```

Reply:
updated